.github/workflows/fire-jenkins.yml

```diff
@@ -0,0 +1,25 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Fire Jenkins
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    # Runs a single command using the runners shell
+    - name: Fire Jenkins
+      run: curl -X POST -d "" -u "$JENKINS_AUTH" "https://builds.spacestation14.io/jenkins/job/SS14%20Content/build?token=$JENKINS_SS14_CONTENT_BUILD_TOKEN&cause=GitHub+actions+push"
+      env:
+        JENKINS_AUTH: ${{ secrets.JENKINS_AUTH }}
+        JENKINS_SS14_CONTENT_BUILD_TOKEN: ${{ secrets.JENKINS_SS14_CONTENT_BUILD_TOKEN }}
```

Comment:
Unless these environment variables are used implicitly by something in the build step, you can inline them in the command and the workflow will substitute them correctly.

Reply:
Hm, I didn't touch that in my PR, github shows that files as not even changed

Reply:
Weird, may have been me misreading a Github notification from a merge or rebase. Ignore this, then.